default_position :
  position:
    x: 0.719485937580114
    y: 0.11426366098174431
    z: 0.9601198065416131
  orientation:
    x: 0.5158333052982221
    y: 0.49760767471961076
    z: 0.4910325096569348
    w: 0.4951663131365716

default_position_joint_states :
  - -2.739824067854837
  - 3.642702249718859
  - -1.4002863496392823
  - 4.042661172635976
  - 5.157545397608206
  - -1.6057263996584847

use_default_position : True

preplace_transformation :
  position:
    x: 0.0
    y: 0.0
    z: 0.05
  orientation:
    x: 0.0
    y: 0.0
    z: 0.0
    w: 1.0
